docs/ui/cloud-getting-started.md

```diff
@@ -212,7 +212,7 @@ What did we do here? Let's break down the command:
 - `-n test-deployment` is an option to specify a name for the deployment.
 - `-q test` specifies a work queue for the deployment. The deployment's runs will be sent to any agents monitoring this work queue.
 
-The command outputs two files: `basic_flow-manifest.json` contains workflow-specific information such as the code location, the name of the entrypoint flow, and flow parameters. `deployment.yaml` contains details about the deployment for this flow.
+The command outputs two files: `basic_flow-manifest.json` contains workflow-specific information such as the code location, the name of the entrypoint flow, and flow parameters. `basic_flow-deployment.yaml` contains details about the deployment for this flow.
```

Comment:
In addition, we don't create the manifest anymore, so perhaps let's update this to something like "The command outputs `basic_flow-deployment.yaml`, which contains"?

Reply:
Done, thanks!